docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -555,11 +555,14 @@ You may have trouble in the configuration if you see Pod statuses like `RunConta
 
 1. **I tried to set `HostPort` on one workload, but it didn't have any effect**.
     The `HostPort` and `HostIP` functionality is available depending on your Pod Network
-    provider. Please contact the author of the Pod Network solution to find out whether
-    `HostPort` and `HostIP` functionality are available.
-
-    If not, you may still use the [NodePort feature of
-    services](/docs/concepts/services-networking/service/#type-nodeport) or use `HostNetwork=true`.
+    provider.
+    
+    - Verified HostPort CNI providers:
```

Comment:
We should have this in the tabs IMO; and not "leak" any other provider info into the text.
@cmluciano can you fix that please?